### PR TITLE
Release v6.2.0

### PR DESCRIPTION
**Pull request by forrestfwilliams**

### CODE REVIEW

File: src/hyp3_sdk/hyp3.py [hunk 1]
```diff
@@ -46,6 +46,20 @@ def __init__(self, api_url: str = PROD_API, username: Optional[str] = None, pass
...
+        self._check_application_status()
+
+    def _check_application_status(self) -> None:
+        help_url = 'https://hyp3-docs.asf.alaska.edu/using/requesting_access'
+        info = self.my_info()
+        if info['application_status'] == 'NOT_STARTED':
+            warnings.warn(f'User {info["user_id"]} has not yet applied for a monthly credit allotment.'
+                          f' Please visit {help_url} to submit your application.')
+        if info['application_status'] == 'PENDING':
+            warnings.warn(f'User {info["user_id"]}\'s request for access is pending review.  For more information, '
```
Comment (jtherrmann) [inline src/hyp3_sdk/hyp3.py:58]:
```suggestion
            warnings.warn(f'User {info["user_id"]}\'s request for access is pending review. For more information, '
```

File: tests/conftest.py [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
 from uuid import uuid4
 
 import pytest
+import requests
 
 from hyp3_sdk import Job
+from hyp3_sdk.hyp3 import HyP3
+
+
+@pytest.fixture(autouse=True)
```
Comment (jtherrmann) [inline tests/conftest.py:14]: What does `autouse=True` do?

**Comment by jtherrmann** [comment]
Has someone tested these changes locally with a user with each of the four application statuses?

**Comment by jacquelynsmale** [comment]
> Has someone tested these changes locally with a user with each of the four application statuses?

I had just tested with the test cases. Do you have example accounts for each of the application statuses or a way to go in and change that status in the AWS Dynamo DB?

**Comment by jtherrmann** [comment]
I get all of the expected warning messages when testing locally.

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**